po/hu.po

```diff
@@ -1619,24 +1830,30 @@ msgid ""
 "Subtasks:\n"
 "%s"
 msgstr ""
+"\n"
+"Részfeladatok:\n"
+"%s"
 
 #: GTG/plugins/send_email/sendEmail.py:61
 #, python-format
 msgid ""
 "\n"
 "Task content:\n"
-"%s"
+""
```

Comment:
Changed source string which you shouldn't do.
https://github.com/getting-things-gnome/gtg/blob/27b04d3c86a7217ccae712ee0b7fa9891f12858c/GTG/plugins/send_email/sendEmail.py#L61
Note that the translated version also lacks the "%s" part.

Reply:
Huhh. Thanks. Nice catch.

Reply:
Fixed. (Not other similar was found.)